packer/vagrant-box.json

```diff
@@ -1,14 +1,20 @@
 {
     "variables": {
-        "version": "2.0.6",
-        "vm_name": "scf-vagrant-{{isotime \"20060102-1504\"}}",
+        "version": "2.0.6+{{isotime \"20060102-1504\"}}",
```

Comment:
If we change the version format, wouldn't the Vagrantfile need to change also? Right now we can just reference the images which have already been built, sure, but when the box moves to a new version, we'd need to put the generated date component in the box URL manually. I'd prefer to add the date component directly into `image_version` in the triton builder configuration.